.rubocop.yml

```diff
@@ -23,7 +24,7 @@ Layout:
 Metrics:
   Enabled: false
 
-Style:
+Style/TrailingCommaInArguments:
```

Comment:
This leads to a massively larger TODO file. Please keep all style changes for a separate PR.